app/src/main/java/org/mozilla/focus/browser/BrowserNavigationOverlay.kt

```diff
@@ -135,7 +136,16 @@ class BrowserNavigationOverlay @JvmOverloads constructor(
                     onNavigationEvent?.invoke(NavigationEvent.LOAD_TILE, urlStr, null)
                 }
             }
-        }, onTileLongClick = openHomeTileContextMenu)
+        }, onTileLongClick = openHomeTileContextMenu, onTileFocused = {
+            val prefInt = PreferenceManager.getDefaultSharedPreferences(context).getInt(SHOW_UNPIN_TOAST_COUNTER_PREF, 0)
+            if (prefInt < 2) {
```

Comment:
This 2 should be a constant so that it's named, like MAX_UNPIN_TOAST_COUNT, rather than just a random 2 in the code.